docs/core/tools/dotnet-nuget-push.md

```diff
@@ -133,8 +135,17 @@ The `dotnet nuget push` command pushes a package to the server and publishes it.
   > To fix this, upgrade your SDK version or run the following command instead:
   > `dotnet nuget push **/*.nupkg`
 
-- Pushes all *.nupkg* files even if a 409 Conflict response is returned by an HTTP(S) server:
+- Push all *.nupkg* files even if a 409 Conflict response is returned by an HTTP(S) server:
 
   ```dotnetcli
   dotnet nuget push *.nupkg --skip-duplicate
   ```
+
+- Push all *.nupkg* files in the current directory to a local feed directory:
+
+  ```dotnetcli
+  dotnet nuget push *.nupkg -s c:\mydir
+  ```
+
+  This command doesn't store packages in a hierarchical folder structure, which is recommended to optimize performance. For more information, see [Local feeds](//nuget/hosting-packages/local-feeds).
```

Comment:
Hello, the link "Local feeds" seems wrong.

Reply:
You're right, I'll fix it, thanks.